server/utils/urlAccessByStatus.ts

```diff
@@ -0,0 +1,75 @@
+import logger from '../../logger'
+
+const allowedPaths = [
+  {
+    status: 'IN_PROGRESS',
+    allowed: ['/licence/create/.*', '/licence/view/.*'],
+  },
+  {
+    status: 'SUBMITTED',
+    allowed: [
+      '/licence/create/id/(\\d)*/check-your-answers.*',
```

Comment:
As the service design progresses, I wonder if this could catch us out, may be hard to think about every status when adding pages and journeys. Not sure what other ways there are of doing it though..unless just a check on every handler but this feels nice and abstract

Reply:
Its generally just checking the base path of each journey, except check-your-answers and edit, both of which need to be accessed when the licence is in other statuses. I think its pretty clean and follows a pattern. See how it goes?